src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -1015,6 +1023,16 @@ private void WriteDataArchival(DataArchivalConfiguration dataArchival)
                 _context.Write(_writer, dataArchival.ToJson());
             }
 
+            private void WriteTaskConfiguration(IDynamicJson config)
```

Comment:
this method looks generic but is not used in other places, you wanted to apply that elsewhere?